apps/health_alerts/static/js/healthAlertPriorityFilter.js

```diff
@@ -0,0 +1,64 @@
+export default function() {
+  const selectElement = document.querySelector('.select-priority');
+
+  if (selectElement) {
+    // add a handler for the change event for the priority select dropdown
+    selectElement.addEventListener('change', (event) => {
+      const selectedPriority = event.target.value;
+      const allRows = document.querySelectorAll("[data-priority]");
+      // we want to show any row that has the selectedPriority
+      let rowsToShow = document.querySelectorAll(`[data-priority~="${selectedPriority}"]`);
+      if (selectedPriority === '0') {
```

Comment:
I wonder if we could make the choice values more meaningful, like 'all', 'alert', ...